android/run_parallel_test/parallel.conf.js

```diff
@@ -13,21 +13,19 @@ exports.config = {
     name: 'parallel_appium_test',
     build: 'webdriver-browserstack',
     browserName: 'android',
-    app: process.env.BROWSERSTACK_APP_ID || 'bs://<hashed app-id>',
+    app: process.env.BROWSERSTACK_APP_ID || 'bs://43e2a3ac988980aefbb17270c41df964c39a39c0',
```

Comment:
change `bs://43e2a3ac988980aefbb17270c41df964c39a39c01`to 'bs://<hashed app-id>'